columnphysics/icepack_parameters.F90

```diff
@@ -326,6 +326,9 @@ module icepack_parameters
 
       character (len=char_len), public :: &
          wave_spec_type = 'constant'  ! 'none', 'constant', or 'random'
+      
+      character (len=char_len), public :: &
```

Comment:
This isn't super important, but wave_height_type and wave_height_spec could be listed under the same character declaration (here and elsewhere)